tests/api_tests/nodes/test_views.py

```diff
@@ -1675,19 +1677,18 @@ def test_update_project_with_no_title_or_category_formatting(self):
         errors = res.json['errors']
         assert(isinstance(errors, list))
         assert_equal(len(errors), 2)
-        assert('category' in res.json['errors'][0]['meta']['field'])
-        assert('This field is required.' in res.json['errors'][0]['detail'])
-        assert('title' in res.json['errors'][1]['meta']['field'])
-        assert('This field is required.' in res.json['errors'][0]['detail'])
+        assert_equal(res.json['errors'][0]['source'], {'pointer': '/data/attributes/category'})
+        assert_equal(res.json['errors'][0]['detail'], 'This field is required.')
```

Comment:
The line you're referring to is in test_update_project_with_no_title_or_category_formatting